javascript/javascript2/week2/index.js

```diff
@@ -0,0 +1,130 @@
+// Doubling of number
+
+let numbers = [1, 2, 3, 4];
+
+const newNumbers = numbers
+  .filter(function (params) {
```

Comment:
Great, maybe you could improve these methods by using the arrow function syntax that you learned in the last lecture. 
for example:

`.filter(params => params % 2 !== 0) // with arrow function if it is one line you dont need to even use the "return key word and it just returns it, cool no ?`

Reply:
Fixed